packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -161,97 +206,106 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
-                        ? 'add-payment-method__footer'
-                        : 'buy-sell__modal-footer'
-                }
-            >
-                {table_type === buy_sell.SELL && is_account_balance_low && <LowBalanceMessage />}
-                {!!error_message && <BuySellFormError />}
-                {my_profile_store.should_show_add_payment_method_form ? (
-                    <AddPaymentMethodForm formik_ref={formik_ref} should_show_separated_footer={true} />
-                ) : (
-                    <Form
-                        advert={selected_ad}
-                        handleClose={onCancel}
-                        handleConfirm={onConfirmClick}
-                        setIsSubmitDisabled={setIsSubmitDisabled}
-                        setErrorMessage={setErrorMessage}
-                        setSubmitForm={setSubmitForm}
-                        setPageFooterParent={setPageFooterParent}
-                    />
-                )}
-            </MobileFullPageModal>
-        );
-    }
-    if (should_show_popup) {
-        return (
-            <Modal
-                className='buy-sell__modal'
-                height={table_type === buy_sell.BUY ? 'auto' : '649px'}
-                width='456px'
-                is_open={should_show_popup}
-                title={generateModalTitle(formik_ref, my_profile_store, table_type, selected_ad)}
-                portalId={general_store.props.modal_root_id}
-                toggleModal={onCancel}
-            >
-                {/* Parent height - Modal.Header height - Modal.Footer height */}
-                <ThemedScrollbars height={table_type === buy_sell.BUY ? '100%' : 'calc(100% - 5.8rem - 7.4rem)'}>
-                    <Modal.Body className='buy-sell__modal--layout'>
-                        {table_type === buy_sell.SELL && is_account_balance_low && <LowBalanceMessage />}
-                        {!!error_message && <BuySellFormError />}
-                        {my_profile_store.should_show_add_payment_method_form ? (
-                            <AddPaymentMethodForm formik_ref={formik_ref} should_show_separated_footer />
-                        ) : (
-                            <Form
-                                advert={selected_ad}
-                                handleClose={onCancel}
-                                handleConfirm={onConfirmClick}
-                                setIsSubmitDisabled={setIsSubmitDisabled}
-                                setErrorMessage={setErrorMessage}
-                                setSubmitForm={setSubmitForm}
-                            />
-                        )}
-                    </Modal.Body>
-                </ThemedScrollbars>
-                {!my_profile_store.should_show_add_payment_method_form && (
-                    <Modal.Footer has_separator>
-                        {my_profile_store.should_show_add_payment_method_form ? null : (
+    return (
```

Comment:
All the changes below here are just that I'm just wrapping the `MobileFullPageModal` within `MobileWrapper` and `Modal` within `DesktopWrapper` to avoid a bug where both modals are stacking/showing at the same time:
![Screenshot 2022-09-01 at 2 33 00 PM](https://user-images.githubusercontent.com/103016120/187847348-4e3c27c3-4f02-4e53-856c-653d312ce96e.png)